lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -305,17 +308,15 @@ class CategoryRenderer {
       return failedElem;
     }
 
-    const expandable = true;
-    const elements = this._renderGroupedAudits(auditRefs, groupDefinitions, {expandable});
-
     const clumpInfo = this._clumpDisplayInfo[clumpId];
     // TODO: renderAuditGroup shouldn't be used to render a clump (since it *contains* audit groups).
```

Comment:
this TODO makes no sense now. Maybe `// TODO: renderAuditGroup shouldn't be used to render a clump since it's a clump, not a group.`